hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/WritableStoreFactory.java

```diff
@@ -44,4 +45,11 @@ public interface WritableStoreFactory {
      * @return a new {@link WritableTokenRelationStore}
      */
     public WritableTokenRelationStore createTokenRelStore();
+
+    /**
+     * Get a {@link WritableSpecialFileStore}.
+     *
+     * @return a new {@link WritableSpecialFileStore}
+     */
+    public WritableSpecialFileStore createSpecialFileStore();
```

Comment:
null annotation for result

Reply:
This comment is no longer relevant because this method went away due to a separate PR that merged to develop.